x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractAggregationTestCase.java

```diff
@@ -263,13 +262,19 @@ private void aggregateWithIntermediates(Expression expression) {
     }
 
     private void evaluate(Expression evaluableExpression) {
-        Object result;
-        try (var evaluator = evaluator(evaluableExpression).get(driverContext())) {
-            try (Block block = evaluator.eval(row(testCase.getDataValues()))) {
-                result = toJavaObjectUnsignedLongAware(block, 0);
-            }
```

Comment:
This was never being called, it would be failing otherwise. So I moved here the testFold() logic, and now anything getting here must be foldable.
If a case arise where this evaluation is required, we can recover it. But for now it's meaningless